src/tile/index.html

```diff
@@ -0,0 +1,12 @@
+<div class="gb-product-tile">
```

Comment:
@effervescentia this was a first attempt to not rely on tag names for CSS and only rely on BEM classes